NEWS.md

```diff
@@ -1,5 +1,10 @@
 # teal 0.15.2.9039
 
+### New features
+
+* Footer of `teal` apps is enriched with a modal that shows `.lock` file created with `pak` package. 
```

Comment:
this seems old. this feature now using `renv `right? Please update it.

Reply:
oh yes!

Reply:
32dc638fcad93c316093043d5619009274992c37